backend/docs/users.md

```diff
@@ -227,6 +227,7 @@ Input Sample:
   "country": "mm",
   "district": "pabedan",
   "division": "yangon",
+  "street_address": "19 street",
```

Comment:
is it already in master right ? https://github.com/thingaha/thingaha-web/blob/master/backend/docs/users.md